Source/WebCore/platform/graphics/mac/controls/MenuListButtonMac.mm

```diff
@@ -159,50 +162,39 @@ static void drawMenuListBackground(GraphicsContext& context, const FloatRect& re
 
     float leftEdge;
     if (isRightToLeft)
-        leftEdge = bounds.x() + arrowPaddingAfter * style.zoomFactor;
+        leftEdge = logicalBounds.x() + arrowPaddingAfter * style.zoomFactor;
     else
-        leftEdge = bounds.maxX() - arrowPaddingAfter * style.zoomFactor - arrowWidth;
+        leftEdge = logicalBounds.maxX() - arrowPaddingAfter * style.zoomFactor - arrowWidth;
 
     GraphicsContextStateSaver stateSaver(context);
 
     context.setFillColor(style.textColor);
     context.setStrokeStyle(StrokeStyle::NoStroke);
 
-    // Draw the top arrow
-    Vector<FloatPoint> arrow1 = {
+    Vector<FloatPoint> topArrow = {
         { leftEdge, centerY - spaceBetweenArrows / 2.0f },
         { leftEdge + arrowWidth, centerY - spaceBetweenArrows / 2.0f },
         { leftEdge + arrowWidth / 2.0f, centerY - spaceBetweenArrows / 2.0f - arrowHeight }
     };
-    context.fillPath(Path(arrow1));
 
-    // Draw the bottom arrow
-    Vector<FloatPoint> arrow2 = {
+    Vector<FloatPoint> bottomArrow = {
         { leftEdge, centerY + spaceBetweenArrows / 2.0f },
         { leftEdge + arrowWidth, centerY + spaceBetweenArrows / 2.0f },
         { leftEdge + arrowWidth / 2.0f, centerY + spaceBetweenArrows / 2.0f + arrowHeight }
     };
-    context.fillPath(Path(arrow2));
 
-    constexpr auto leftSeparatorColor = Color::black.colorWithAlphaByte(40);
-    constexpr auto rightSeparatorColor = Color::white.colorWithAlphaByte(40);
+    // Rotate the arrows for vertical writing mode.
```

Comment:
nitpick, but comments usually mention _why_ we do things.

```suggestion
    // Rotate the arrows for vertical writing mode since the popup appears on the side of the menulist instead of under it.
```